circle.yml

```diff
@@ -50,13 +50,13 @@ dependencies:
     - docker-compose version
     - docker version
   override:
-    - .circle/docker-compose.sh pull
+    - .circle/docker-compose.sh pull ${DISTRO}
   post:
-    - .circle/docker-compose.sh build
+    - .circle/docker-compose.sh build ${DISTRO}
```

Comment:
And think about this seriously, where you need to iterate over $DISTROS, np. But excessive var usage like build ${DISTRO}, test ${DISTRO} etc is really not required... again in case of docker-compose.sh. About other things i dun know. We'll see and figure it out as soon as the appear on the horizon.